_security/access-control/api.md

```diff
@@ -1297,6 +1297,85 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
+### Check configuration upgrade
```

Comment:
```suggestion
### Check if configuration upgrade is available
```

Reply:
Reorder the title - trying to keep the word count down since its copied around into tables of contents.